lucky-nums/static/lucky.js

```diff
@@ -1,12 +1,45 @@
 /** processForm: get data from form and make AJAX call to our API. */
```

Comment:
Your DOM manipulation is simple enough that you should use vanilla JS instead of jQuery.